java/code/src/com/suse/manager/webui/websocket/VirtNotifications.java

```diff
@@ -193,7 +193,7 @@ public void onMessage(Session session, String messageBody) {
      * @param message the message to be sent
      */
     public static void sendMessage(Session session, String message) {
-        synchronized (session) {
+        synchronized (LOCK) {
```

Comment:
with this change one can only send a message at the same time, in an application, instead of per session. Is this waht you want to archive?